app/grandchallenge/participants/views.py

```diff
@@ -115,3 +129,103 @@ def get_success_url(self):
             "participants:registration-list",
             kwargs={"challenge_short_name": self.object.challenge.short_name},
         )
+
+
+class RegistrationQuestionList(
+    LoginRequiredMixin, PermissionListMixin, ListView
+):
+    model = RegistrationQuestion
+    permission_required = "participants.view_registrationquestion"
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        queryset = queryset.filter(challenge=self.request.challenge)
+        return queryset
+
+    def dispatch(self, request, *args, **kwargs):
+
+        if not self.request.user.has_perm(
+            "challenges.change_challenge",
+            self.request.challenge,
+        ):
+            # View contains admin side-panel
+            raise PermissionDenied
+
+        return super().dispatch(request, *args, **kwargs)
+
+
+class RegistrationQuestionCreate(
+    LoginRequiredMixin,
+    SuccessMessageMixin,
+    ObjectPermissionRequiredMixin,
+    CreateView,
+):
+    model = RegistrationQuestion
+    form_class = RegistrationQuestionCreateForm
+
+    success_message = "Question successfully created"
+
+    permission_required = "challenges.add_registration_question"
+
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
```

Comment:
Code style: 3 new views, 3 different ways of spacing the attributes, 3 different ways of ordering them. Please be consistent.